src/NClient.Tools/NClient.DotNetTool/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using CommandLine;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NClient.CodeGeneration.Abstractions;
+using NClient.CodeGeneration.Interfaces.NSwag;
+using NClient.DotNetTool.Loaders;
+
+namespace NClient.DotNetTool
+{
+    // ReSharper disable once ClassNeverInstantiated.Global
+    internal class Program
+    {
+        public static async Task<int> Main(string[] args)
+        {
+            var serviceProvider = BuildServiceProvider();
+
+            return await Parser.Default.ParseArguments<CommandLineOptions>(args)
+                .MapResult(async opts =>
+                    {
+                        var logger = serviceProvider.GetRequiredService<ILoggerFactory>().CreateLogger<Program>();
+                        try
+                        {
+                            var specification = await serviceProvider.GetRequiredService<ILoaderFactory>().Create(opts).Load();
+                            var result = await serviceProvider.GetRequiredService<FacadeGenerator>().GenerateAsync(opts, specification);
+                            await Save(opts, result);
```

Comment:
I would want to see messages that the operation was successfully and where the facades were saved :)

Reply:
Done.